framework/src/controller/helpers/validator/keywords/formatters.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+	stringToIpPortSet: value => {
+		if (typeof value === 'string') {
+			return value.split(',').map(peer => {
+				const [ip, wsPort] = peer.split(':');
+				return {
+					ip,
+					wsPort: wsPort || 5000,
+				};
+			});
+		}
+		return [];
+	},
+
+	stringToDelegateList: value => {
```

Comment:
i think this is not framework logic

Reply:
Chain module uses `LISK_FORGING_DELEGATES` env variable which can only be specified as string. So this logic is responsible to convert string to delegate list.